modules/installation-images-samples-disconnected-mirroring-assist.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-restricted-networks-preparations.adoc
+// * openshift_images/samples-operator-alt-registry.adoc
+// * openshift_images/configuring-samples-operator.adoc
+
+[id="installation-images-samples-disconnected-mirroring-assist_{context}"]
+= Cluster Samples Operator assistance for mirroring
+
+During installation, {product-title} creates a config map named `imagestreamtag-to-image` in the `openshift-cluster-samples-operator` namespace. The `imagestreamtag-to-image` config map contains an entry, the populating image, for each image stream tag.
+
+The format of the key for each entry in the data field in the config map is `<imagestreamname>_<imagestreamtagname>`.
```

Comment:
What do you think about adding underscores for the names for readability?
>`<image_stream_name>_<image_stream_tag_name>`

Reply:
Makes sense to me!